pkgs/development/python-modules/ansible/core.nix

```
@@ -105,6 +105,8 @@ buildPythonPackage rec {
     description = "Radically simple IT automation";
     homepage = "https://www.ansible.com";
     license = licenses.gpl3Plus;
-    maintainers = [ ];
+    maintainers = [
```

```suggestion
    maintainers = with lib.maintainers; [
```

The following change should unbreak eval (Also for the other file).

---

Thank you for your response, I've seen the issue and fixed it independently, thus I looked up other pkgs and used only `with maintainers;` – I hope this is ok.